README.md

```diff
@@ -22,7 +22,7 @@ To learn more about openBalena, visit [balena.io/open][open-balena-website].
 - **Container-based**: Benefit from the power of virtualization, optimized for the edge
 - **Scalable**: Deploy and manage one device, or one million
 - **Powerful API & SDK**: Extend openBalena to fit your needs
-- **Built-in VPN**: Access your devices regardless of their network environment
+- **Built-in terminal connection**: Access your devices regardless of their network environment
```

Comment:
I think “VPN” here actually makes more sense — given the subtitle, I personally read it literally as “virtual private network” which is in fact accurate. Not sure how this can be improved.

Reply:
This is a feature list. But the exposed feature for the end-user is not VPN, but rather an ability to establish specific types of connections with the devices (call supervisor HTTP endpoint or establish an SSH connection with the device). The end-user cannot work with the VPN directly. The fact we use a VPN server is only an implementation detail that potentially can be changed without changing the features we expose.

Does it make sense?

Reply:
I get that. This is indeed a feature list but I think “terminal connection” is just not accurate enough or applicable given the feature description. I can’t come up with something better but if you feel strongly about removing the VPN reference I think the whole line should be rewritten.

Reply:
We should also consider that the exposed VPN functionality (for want of a better term!) is not limited to terminals. Public URLs depend on the VPN service, as does the `balena tunnel` functionality.

Reply:
I agree "terminal connection" is a bad phrase. Thinking about a better variant.

Reply:
It seems like this PR was opened on a whim, and I'm not sure a code review is the best place for the resultant discussion. Would we be better closing/shelving this PR for now and taking it to a product(?) call?

Reply:
I'm just opening a discussion. It makes sense to me to talk about this on a product call next week.

Reply:
Cool, I think that would be best. FWIW “the tunnel service” is what the user-facing service on the VPN is colloquially known as (the HTTP CONNECT proxy we expose allowing TCP connections to arbitrary device ports).